lib/wallet/test/integration/shelley-integration-test.hs

```diff
@@ -359,17 +363,22 @@ specWithServer testDir (tr, tracers) = aroundAll withContext
                     , Cluster.cfgNodeLogging = LogFileConfig Info Nothing Info
                     , Cluster.cfgClusterDir = Tagged @"cluster" testDir
                     , Cluster.cfgClusterConfigs = cfgClusterConfigs
+                    , Cluster.cfgTestnetMagic = testnetMagic
                     }
             withCluster tr' clusterConfig faucetFunds
                 $ onClusterStart (onReady $ T.pack smashUrl) dbDecorator
 
     tr' = contramap MsgCluster tr
 
+    testnetMagic = Cluster.TestnetMagic 42
```

Comment:
Nitpick: I'm guessing you should get the type-level testnet magic from `main` rather than redefining it here.